docs/src/constraints.md

```diff
@@ -235,6 +235,28 @@ The constraint can also be retrieved from its name using
 constraint_by_name
 ```
 
+## Start Values
+
+There is a way to assign a starting value to some constraint's dual (for warmstarting the optimizer) for each constraint: using [`set_dual_start_value`](@ref).
+
+The start value of a constraint's dual can be queried using [`dual_start_value`](@ref). If
+no start value has been set, [`dual_start_value`](@ref) will return `Nothing`.
+
+```jldoctest constraint_dual_start; setup=:(model=Model())
+julia> @variable(model, x)
+x
+
+julia> @constraint(model, con, x >= 10)
+con : x ≥ 10.0
+
+julia> dual_start_value(con)
+
+julia> set_dual_start_value(con, 2)
+
+julia> dual_start_value(con)
+2.0
+```
+
```

Comment:
What about explaining how to transfer an optimal dual solution to a dual warm start? This is not as easy to do as with variables.

Reply:
Shouldn't it be trivial since we are using shapes?
`sol = dual(con)`
`sol` would be in the appropriate shape (scalar, vector, matrix, etc), which depends on the constraint type.
Then
`set_dual_start_value(con, sol)` should just work since a value in the proper shape is expected.

Reply:
Now think you meant this due `list_of_constraint_types`

Reply:
Yeah, I meant to mention the equivalent of:
```
set_start_value.(all_variables(model), value.(all_variables(model)))
```
We discuss this at https://jump.dev/JuMP.jl/v0.21.5/variables/#Start-values-1, so we should do the same for constraints.